src/code-examples/disclosure/javascript.js

```diff
@@ -14,8 +16,7 @@ const javascript = `document.addEventListener("click", (event) => {
     //Toggles target button element.
     event.target.setAttribute("aria-expanded", !isExpanded);
     //Toggles element to show or hide (text).
-    disclosureTarget.toggleAttribute("hidden", isExpanded);
-  }
+    disclosureTarget.classList.toggle("is-hidden");
```

Comment:
Oh I didn't see that it was originally using `hidden`. I think I paired with Terri on this, which is why my preference was already there 😂